Add Dex server selector, fix selector not scrollable

song50119 opened the pull request:
Fix #1010 
Resolves #1009 

<img width="799" alt="image" src="https://user-images.githubusercontent.com/27733432/181350877-5301e688-e362-4697-8ceb-186061a1da66.png">

Commit:
Fix wallet selector page not scrollable

Sirmorrison reviewed (changes requested):
The entire page scrolls which should not be.. the title and create wallet button should not scroll.

Commit:
Add Dex server selector
- List available Dex server from dcrdex module
- Ability add custom dex server, pay the fee and save dex server to db to be reuse in next session
- Change name wallet_list_page.go -> wallet_dex_selector_page.go. Implement components.WalletSelector, components.DexServerSelector to WalletDexServerSelector struct

itswisdomagain commented on ui/page/dexclient/add_dex_modal.go:
```diff
@@ -352,3 +235,17 @@ func confirmRegisterModalDesc(dexServer *core.Exchange, selectedFeeAsset string)
 	return fmt.Sprintf("Confirm DEX registration. When you submit this form, %s will be spent from your wallet to pay registration fees.", feeAmt)
 }
+
+// saveDexServer after pay the fee success save the host and cert to db.
+func (md *AddDexModal) saveDexServer(host string, cert []byte) {
```
Why are these details saved here after successful fee payment? Doesn't the official dex client library handle this?

itswisdomagain reviewed (changes requested):
The debug setting option to reset the DEX client is no longer available. I'm unable to properly test this PR.

Also, on app load, the select wallet page also prompts to select a DEX server. Not sure what the idea for that is. Do you need to select a wallet AND a DEX server? Can you proceed by just selecting a DEX server but not selecting a wallet? Where page does that lead to?

Sirmorrison committed:
Update UI to updated layout.
- link settings button to settings page
- fix wallet selector sync bug
- clean up and removed redundant lines of code.

dreacot reviewed (approved):
LGTM
